src/starknetCli.ts

```diff
@@ -28,15 +28,9 @@ const MAX_FEE = 'max_fee';
 const NETWORK = 'network';
 const WALLET = 'wallet';
 
-const warpVenvPrefix = `PATH=${path.resolve(__dirname, '..', 'warp_venv', 'bin')}:$PATH`;
-const CAIRO1_COMPILE_BIN = path.resolve(
-  __dirname,
-  '..',
-  'cairo1',
-  getPlatform(),
-  'bin',
-  'starknet-compile',
-);
+export const BASE_PATH = path.dirname(__dirname);
+const warpVenvPrefix = `PATH=${path.resolve(BASE_PATH, 'warp_venv', 'bin')}:$PATH`;
```

Comment:
Is there any specific reason why we call `resolve` here instead of `join`?

Reply:
I think it is because resolve gives the absolute path no matter the arguments

Reply:
Is BASE_PATH relative path then?  if not it can be `join` as well

Reply:
Nop, used `join`